src/views/EditSidebar.vue

```diff
@@ -80,6 +91,26 @@ export default {
 		subtitle() {
 			return this.isLoading ? 'LOADING' : 'LOADED'
 		},
+		location() {
```

Comment:
You have two options. Either you use a computed property with get(){} and set(value){} and set the new value inside set.
Or you add `@change="setNewValue"` on the input element and create a new setNewValue method,